.travis.yml

```
@@ -1,5 +1,5 @@
 sudo: false
 language: ruby
 rvm:
-  - 2.2.5
-before_install: gem install bundler -v 1.12.5
+  - 2.6.6
```

```suggestion
  - 2.5.8
  - 2.6.6
  - 2.7.2
```
давай еще парочку добавим, которые на данный момент актуальный

---

насчет 2.7 не уверен, но давай попробуем